Add Event hooks docs in console (#46)
calimero-is-near/docs: docs/getting_started/event_hooks.md

Reviewer: Move this file to a separate directory

title: Event Hooks
sidebar_position: 9
---

> Event hooks service will receive all events from near-lake stream. It will then check defined webhooks and based on filters either call the destination service or not. Destination service is the user defined service whose url has a user set while creating a webhook. That service may have an open endpoint or require an auth token for access.

Reviewer: ```suggestion
Event hooks service will receive all events from the near-lake stream.
It will then check the defined webhooks and based on filters either call the destination service or not. Destination service is the user defined service whose URL is set when creating a webhook.
That service may have an open endpoint or require an auth token for access.
```

> User can specify with a webhook authorization mechanism so the service can authorise incoming data.

Reviewer: ```suggestion
User can add an authorization token when creating a webhook.
```
